lib/web/imageRouter/index.js

```diff
@@ -15,12 +16,25 @@ const errors = require('../../errors')
 const imageRouter = (module.exports = Router())
 
 async function checkUploadType (filePath) {
-  const typeFromMagic = await FileType.fromFile(filePath)
+  const extension = path.extname(filePath).toLowerCase()
+  let typeFromMagic = await FileType.fromFile(filePath)
+  if (extension === '.svg') {
+    const fileContent = fs.readFileSync(filePath)
+    if (isSvg(fileContent)) {
+      typeFromMagic = {
+        ext: 'svg',
+        mime: 'image/svg+xml'
+      }
+    }
+  }
   if (typeFromMagic === undefined) {
     logger.error('Image upload error: Could not determine MIME-type')
     return false
   }
-  if (path.extname(filePath) !== '.' + typeFromMagic.ext) {
+  if (extension === '.jpeg' && typeFromMagic.ext === 'jpg') {
+    typeFromMagic.ext = 'jpeg'
+  }
```

Comment:
There do exist even more "jpeg" file extensions like .jpe or .jfif. [[1]]

If we drop the check (see my previous comment) this would become unnecessary.

[1]: https://de.wikipedia.org/wiki/JPEG_File_Interchange_Format

Reply:
I'm not to sure that those formats are widely used, but I can add checks for those, too.

Reply:
Here are my test files.

Curiously `convert acid.svg acid.jfif` did not produce a correct `.jfif` file but some sort of svg…

[jpeg.zip](https://github.com/hedgedoc/hedgedoc/files/6222256/jpeg.zip)

Reply:
okay